test/extensions/stats_sinks/dog_statsd/config_test.cc

```diff
@@ -122,7 +121,7 @@ TEST_P(DogStatsdConfigLoopbackTest, DefaultBufferSize) {
 }
 
 TEST_P(DogStatsdConfigLoopbackTest, WithCustomPrefix) {
-  const std::string name = StatsSinkNames::get().DogStatsd;
+  const std::string name = "envoy.stat_sinks.dog_statsd";
```

Comment:
we're going to share all these instances of the same string in constant in a header file, right?